ui-manchette-with-spacetimechart/src/utils/vector.ts

```diff
@@ -0,0 +1,11 @@
+export type Point = {
```

Comment:
use the same name for the file and the type.

Reply:
I think that the type is really used to describe a point and not a Vector, so I would keep this name

Reply:
yes it is !

Reply:
use **point** for both then, if vector doesn’t exist

Reply:
done